projects/ThinktectureIdentityServer.md

```diff
@@ -1,21 +1,21 @@
-# Thinktecture.IdentityServer
+IdentityServer

```

Comment:
@leastprivilege - looks like it's missing the heading tag. Should be "# IdentityServer". Can you fix?